mos-platform/nes/rompoke/rompoke-table.ld

```diff
@@ -0,0 +1,1028 @@
+/* Automatically generated by generate-rompoke-table-linkscript.lua. */
+SECTIONS {
+  .rom_poke_table_0 : {
+    __rom_poke_table = .;
+    FILL(0x00000000)
+    . = . + (__rom_poke_table_size > 0 ? 1 : 0);
+  } >c_readonly
+  .rom_poke_table_1 : {
+    FILL(0x01010101)
```

Comment:
Does `FILL(0x01)` not work? If not, add a comment to specify this; the fact that these are 32-bits each is very suspicious and could use a rationale.

Reply:
Works just fine; it was a leftover.